pandapipes/component_models/abstract_models/branch_models.py

```diff
@@ -28,6 +28,22 @@
 
 class BranchComponent(Component):
 
+    @classmethod
+    def table_name(cls):
+        return NotImplementedError
+
+    @classmethod
+    def get_component_input(cls):
+        return NotImplementedError
+
+    @classmethod
+    def get_result_table(cls, net):
+        return NotImplementedError
+
+    @classmethod
+    def from_to_node_cols(cls):
+        raise NotImplementedError
+
```

Comment:
Yes, this is very important, as we are using generic functions, where the column names are not defined, but inquired from the component model. I.e. you might define your own branch component that uses "junction1" and "junction2" instead of "from_junction" and "to_junction".

Reply:
Okay, I wasn't that deep in that topic that I understood everthing in detail. Thanks for the clarification :). We should keep it here then.